src/Components/Shared/Button/button.module.css

```diff
@@ -0,0 +1,65 @@
+.delete {
+  margin: 0 5px;
+  padding: 2px 10px 0px 10px;
+  min-height: 30px;
```

Comment:
The buttons are too small, please add a little more width and height, also add a font size bigger

Reply:
I think this measures are better for a button
   width: 150px;
   height: 50px;
   font-size: 16px;